Validation/RecoTrack/python/PostProcessorTracker_cfi.py

```diff
@@ -15,8 +15,13 @@ def _addNoFlow(module):
         if not tmp[ind-1] in _noflowSeen:
             module.noFlowDists.append(tmp[ind-1])
 
+listOfDirs = cms.untracked.vstring("Tracking/Track/*", "Tracking/TrackTPPtLess09/*", "Tracking/TrackFromPV/*", "Tracking/TrackFromPVAllTP/*", "Tracking/TrackAllTPEffic/*", "Tracking/TrackBuilding/*", "Tracking/TrackConversion/*", "Tracking/TrackGsf/*", "Tracking/TrackBHadron/*")
+
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+phase2_tracker.toModify(listOfDirs.extend(["Tracking/TrackTPEtaGreater2p7/*"]))
```

Comment:
This line doesn't do what you expect. It will always extend `listOfDirs` (and the return value of `extend()` is then passed to `toModify()`).

Reply:
My approach was that I just give the list of all possible directories here. It leads to DQMGenericClient to print warnings on missing directories, but it nevertheless does the job for those directories that exist. This approach allows a single harvesting configuration to cover all use cases and thus a script to run the (tracking-only) harvesting `harvestTrackValidationPlots.py`.

Reply:
thanks @makortel 
I hope the updated version will make the job